Rakefile

```diff
@@ -1,20 +1,19 @@
 require 'rspec/core/rake_task'
 require 'rubygems/package_task'
+require 'mixlib/shellout/version'
 
-Dir[File.expand_path("../*gemspec", __FILE__)].reverse.each do |gemspec_path|
-  gemspec = eval(IO.read(gemspec_path))
+Dir[File.expand_path("../*gemspec", __FILE__)].each do |gemspec_path|
+  gemspec = eval(File.read(gemspec_path))
   Gem::PackageTask.new(gemspec).define
 end
 
-require 'mixlib/shellout/version'
-
 desc "Run all specs in spec directory"
 RSpec::Core::RakeTask.new(:spec) do |t|
   t.pattern = FileList['spec/**/*_spec.rb']
 end
 
 desc "Build it and ship it"
-task :ship => [:clean, :gem] do
+task ship: [:clobber_package, :gem] do
```

Comment:
Yep.

C:\Users\ksubramanian\repo\chef\mixlib-shellout [ksubrama/win64]> bundle exec rake clean
rake aborted!
Don't know how to build task 'clean'

(See full trace by running task with --trace)
C:\Users\ksubramanian\repo\chef\mixlib-shellout [ksubrama/win64]> bundle exec rake -T
rake clobber_package  # Remove package products
rake gem              # Build the gem file mixlib-shellout-2.0.1.gem / Buil...
rake package          # Build all the packages
rake repackage        # Force a rebuild of the package files
rake ship             # Build it and ship it
rake spec             # Run all specs in spec directory

There is no Gemfile.lock in this repo - so it just grabs a recent rake.